client/index.html

```diff
@@ -12,7 +12,8 @@
     <div id="image">
```

Comment:
General comment about index.html file:
- Quite hard to read the code as it's not indented as standard. 
Apart from that, all good here!